fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportTransientUserData.java

```diff
@@ -6,7 +6,9 @@
 
 package com.ibm.fhir.bulkimport;
```

Comment:
looking at these package names, should we have a single parent package for bulkdata, and call out each feature underneath? establishing it as a hierarchy? just a thought... no action needed, just a design question

Reply:
good question,  I thought this project is for bulkdata webapp, so I didn't add a bulkdata as part of the package name. and we can change it whenever we want in the future.

Reply:
yeah - no change necessary - as I see these Java classes again, I see new things :)